packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -56,6 +57,7 @@ const GOERLI = {
   nativeTokenName: 'Ethereum',
   name: 'Goerli',
   relayServiceURL: 'https://relay-goerli.staging.stack.cards/api',
+  rpcUrl: 'https://rpc.ankr.com/eth_goerli',
```

Comment:
can we get the RPC URL from hub config?

Reply:
Is there a reason why that would be better? I'd prefer if the sdk wouldn't rely on the hub to get this config

Reply:
I think we can follow the same approach that is used by other SDK functions, instead of initializing the provider in the SDK (in this case in your `currentGasPrice` function) we can let the client initialize the provider and pass it as a parameter to `currentGasPrice(provider: JsonRPCProvider, network: string, gasTokenAddress: string)`. In this way, the SDK doesn't need to care about the RPC url.

Reply:
That makes sense. Thanks, will adjust this

Reply:
Fixed